File: Week 1 - C/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int x;
    do
    {
        x = get_int("Choose a number between 1 and 8\n");
    }
    while (x < 1 || x > 8);
    for (int i = 0; i < x; i++)
    {
        for (int j = 0; j < x - i - 1; j++)
        {
            printf(" ");
        }

        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }

        printf("  ");

        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
    }
}

File: Week 6 - Python/sentimental-mario-more/mario.py
from cs50 import get_int

height = get_int("Height : ")
while height < 1 or height > 8:
    height = get_int("Height : ")

for h in range(height):
    for i in range(height - h - 1):
        print(" ", end="")
    for j in range(h + 1):
        print("#", end="")
    print("  ", end="")
    for j in range(h + 1):
        print("#", end="")
    print("")
